_data/fr/navigation.yml

```diff
@@ -0,0 +1,28 @@
+nav:
+- title:                 Utiliser Jamulus
+  subfolderitems:
+    - page:              Pour commencer
+      url:               /wiki/Getting-Started
+      subsubfolderitems:
+        - page:          Windows
+          url:           /wiki/Installation-for-Windows
+        - page:          Macintosh
+          url:           /wiki/Installation-for-Macintosh
+        - page:          GNU/Linux
+          url:           /wiki/Installation-for-Linux
+    - page:              Manuel du logiciel
+      url:               /wiki/Software-Manual
+    - page:              Problèmes potentiels
+      url:               /wiki/Client-Troubleshooting
+- title:                 Plus
+  subfolderitems:
+    - page:              Installer un serveur
+      url:               /wiki/Running-a-Server
+    - page:              Configuration
+      url:               /wiki/Command-Line-Options
+    - page:              Trucs et astuces
+      url:               /wiki/Tips-Tricks-More
+    - page:              Démonstrations
+      url:               /wiki/Demos
+    - page:              Déclarations sur la vie privée
```

Comment:
For this, I've used « Confidentialité » for the menu entry and « Confidentialité et vie privée » for the title of the page.